weave-js/src/components/Panel2/ChildPanelExportReport/ChildPanelExportReport.tsx

```diff
@@ -65,7 +75,11 @@ export const ChildPanelExportReport = ({
           <Button
             icon="add-new"
             className="w-full"
-            disabled={selectedReport == null}
+            disabled={
+              selectedReport == null ||
+              (selectedReport.name === NEW_REPORT_OPTION &&
+                selectedProject == null)
```

Comment:
Okay here's my dilemma:
Right now, if a user selects an existing report we don't need to `setSelectedProject` at any point because (1) the selector doesn't pop up and (2) we can grab that value from the reports object. If a user selects `New report`, then they are forced to select a project which is how I see `setSelectedProject` acting as. Any thoughts on best pattern to go with here?

```
  // Get list of reports across all entities and projects
  const reportsMetaNode = w.opMap({
    arr: w.opEntityReports({
      entity: selectedEntityNode,
    }),
    mapFn: w.constFunction({row: 'report'}, ({row}) => {
      return w.opDict({
        id: w.opReportInternalId({report: row}),
        name: w.opReportName({report: row}),
        updatedAt: w.opReportUpdatedAt({report: row}),
        projectName: w.opProjectName({ <-- WE CAN GET PROJECT NAME WITHIN THEIR SELECTED REPORT
          project: w.opReportProject({report: row}),  
        }),
      } as any);
    }),
  });
  const reports = useNodeValue(reportsMetaNode ?? w.voidNode(), {
    skip: entities.loading,
  });
```

Reply:
I think we can just set selected project when an existing report is selected -- by selecting that report, the user is implicitly choosing the project it belongs to. Doing this keeps the data source consistent regardless of whether user selected new or existing report.

I see where you're coming from by setting `selectedProject` explicitly for the project selector, but I'm just not sure if there's really a reason we need that separation.